app/views/layouts/application.html.erb

```diff
@@ -59,6 +59,7 @@
 
     <%= tag.meta id: 'data-discourse-setup', data: client_side_setup_data %>
 
+    <meta name="discourse/config/environment" content="<%=u discourse_config_environment %>" />
```

Comment:
Out of curiosity, why do we need to encode the json string here as a URL?

Reply:
That's how Ember CLI does it and I wanted to keep the format the same.